src/components/screen/Oauth/Oauth.vue

```diff
@@ -99,6 +102,10 @@ export default {
       return !this.isInited || this.isLoadingContent;
     },
 
+    isServerSwitcherVisible() {
+      return !ENV.VUE_APP_IS_PRODUCTION;
```

Comment:
`VUE_APP_IS_PRODUCTION` is `true` for staging.
solution is not working correctly.

maybe add `VUE_APP_IS_STRICT` flag and set value `true` for prod and staging env files
then `VUE_APP_IS_PRODUCTION` enable only for prod env file.

Reply:
@dudiq is `VUE_APP_IS_PRODUCTION` affects something in `staging`?

Reply:
try to find usage of this const
and yes, for stagin this const is `true`

Reply:
support your idea with `VUE_IS_STRICT`

Reply:
done